app/models/shipment.rb

```diff
@@ -1,7 +1,11 @@
 class Shipment < ActiveRecord::Base
   belongs_to :order
 
-  enum shipping_carrier: { FedEx: 0, USPS: 1, UPS: 2 }
+  enum shipping_carrier: { FedEx: 0, USPS: 1, UPS: 2, Hand: 3 }
+
+  def self.valid_carriers
+    shipping_carriers.select { |k, _v| %w(FedEx USPS UPS).include? k }
```

Comment:
I think this might be a tad nicer: http://api.rubyonrails.org/classes/Hash.html#method-i-slice

I can try it later and just merge if it works